scripts/calendar/mongolian_generator.py

```
@@ -0,0 +1,204 @@
+    t = (24 * (y - epoch) - beta) % 65
+    return t >= 41
+
+
+def true_month(y, m, is_leap):
+    p = 67 * mstar(y, m) + betastar
+    ix = p % 65
+    pp = (p - ix) // 65
+    return pp if is_leap or ix < ixx else pp + 1
+
+
+def moon_tab(i):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)